go/libcmdline/cmdline.go

```diff
@@ -543,6 +553,11 @@ func (p *CommandLine) PopulateApp(addHelp bool, extraFlags []cli.Flag) {
 			Name:  "debug, d",
 			Usage: "Enable debugging mode.",
 		},
+		cli.BoolFlag{
+			Name:  "disable-ssl-pinning",
+			Usage: "Disable SSL pinning within the app. WARNING: This reduces the security of the app. Do not use " +
```

Comment:
Maybe say when it should be used?  As in behind an invasive proxy?

Reply:
Done